apps/web-mzima-client/src/app/shared/components/collections/collections.component.ts

```diff
@@ -82,6 +82,17 @@ export class CollectionsComponent extends BaseComponent implements OnInit {
     this.featuredEnabled = permissions ? permissions.split(',').includes('Manage Posts') : false;
   }
 
+  @HostListener('keydown', ['$event'])
+  onKeydown(event: KeyboardEvent, collection: CollectionResult) {
+    if (event.key === 'Enter' && event.target instanceof HTMLElement) {
```

Comment:
This is not the review comment for the PR, just me appreciating the "work of art" on the concept and approach generally.
Hmm... This actually looks like some good code you can save somewhere for later @kivuvarosekivuvan
#
> **Extra note:** You may also want to involve the event keycode of the enter key (or which ever key you are doing something like this for).